source/content/guides/agcdn/03-agcdn-wafio.md

```diff
@@ -25,6 +25,12 @@ Support for XSS vulnerabilities.
 
 Image Optimization (IO) renders images using high-density points of presence (POPs) that cache content for longer times and serve millions of image variations. Serving from the edge in this way offloads work from your servers and improves your site's performance. IO supports several formats, including animated GIFs.
 
+<Alert title="Note"  type="info" >
+
+Pantheon partners with [Fastly](https://www.fastly.com/), if you prefer to use a CDN outside of the Pantheon platform. Fastly requires permissions to add domains to accounts, if it detects that a domain already exists.
```

Comment:
This sounds like this is try to cover "Domain Delegation". 

Domain Delegation is a process where Fastly's platform detects if a domain is already used by another account. When it does, it blocks use of the domain and requires a "verification" process (usually via TXT records) for Fastly to allow use of a specific domain. 

However, it is only applicable to **Legacy AGCDN**. This won't be applicable to new SKUs moving forward.

So maybe we don't need to add this here, and place the topic of Domain Delegation somewhere else.

Reply:
Hi @eabquina - I moved the alert to the Global CDN page, if you could take a look!